selfdrive/car/nissan/values.py

```diff
@@ -30,21 +30,21 @@ class CAR:
   ALTIMA = "NISSAN ALTIMA 2020"
 
 
-NISSAN_HARNESS_PARTS = [HarnessPart.harness_box, HarnessPart.rj45_cable, HarnessPart.long_obdc_cable, HarnessPart.usbc_coupler]
+NISSAN_PARTS = [CarPart.harness_box, CarPart.rj45_cable_7ft, CarPart.long_obdc_cable, CarPart.usbc_coupler, CarPart.mount, CarPart.right_angle_obd_c_cable_1_5ft]
 
 
 @dataclass
 class NissanCarInfo(CarInfo):
   package: str = "ProPILOT Assist"
-  harness_kit: HarnessKit = HarnessKit(Harness.nissan_a, parts=NISSAN_HARNESS_PARTS)
+  car_parts: CarParts = CarParts.default([CarPart.nissan_a], default=NISSAN_PARTS)
```

Comment:
now you can get rid of default argument and do something like
```suggestion
  car_parts: CarParts = CarParts([CarPart.nissan_a] + NISSAN_PARTS)
```